packages/yourloops/lib/team/hook.tsx

```diff
@@ -252,56 +217,21 @@ function TeamContextImpl(): TeamContext {
 
   const joinTeam = async (teamId: string): Promise<void> => {
     await TeamApi.joinTeam(teamId, user.id)
-    refresh(true)
+    refresh()
   }
 
-  const initHook = (): void => {
-    if (initialized || lock || !notificationHook.initialized) {
-      return
+  React.useEffect(() => {
+    if (!initialized && notificationHook.initialized) {
+      log.info('init')
```

Comment:
what do you think about removing theses logs, (useless I think)

Reply:
I agree

Reply:
Could you remove all logs in this file (I just discover logs in other methods)